raster/src/main/scala/geotrellis/raster/GridBounds.scala

```diff
@@ -75,9 +75,13 @@ object GridBounds {
 case class GridBounds(colMin: Int, rowMin: Int, colMax: Int, rowMax: Int) {
   def width = colMax - colMin + 1
   def height = rowMax - rowMin + 1
-  if(width.toLong * height.toLong > Int.MaxValue) { sys.error(s"Cannot construct grid bounds of this size: $width x $height") }
-  def size = width * height
-  def isEmpty = size == 0
+
+  def size: Int = width * height
```

Comment:
I tried to use the `@migration` annotation which is mentioned in the official Scaladoc docs, but it doesn't compile. There's no mention of it in the main `scala` repo.